firmware/application/external/app_manager/main.cpp

```diff
@@ -0,0 +1,84 @@
+/*
+ * Copyright (C) 2024 Bernd
```

Comment:
Wrong copyright ;-)

Reply:
Thanks! fixed with he's full name XD
since it's not my code so i won't take credit from it